app/gateway/traffic-control/proxy.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Proxying with {{site.base_gateway}}
+
+description: "Proxying is when {{site.base_gateway}} matches an HTTP request with a [registered route](/gateway/entities/route/) and forwards the request."
+
+content_type: reference
+layout: reference
+
+products:
+  - gateway
+
+related_resources:
+  - text: Route entity
+    url: /gateway/entities/route/
+  - text: Router Expressions language
+    url: /gateway/routing/expressions/
+  - text: Expressions repository
+    url: https://github.com/Kong/atc-router
+
+breadcrumbs:
+  - /gateway/
+---
+
+{{ page.description }} This page details information about how {{site.base_gateway}} handles proxying. The following diagram shows how proxying is handled by {{site.base_gateway}}:
+
+{% mermaid %}
```

Comment:
For reviewers: I used ChatGPT help on this diagram, but I revised everything and the input I provided it was the ordered list below this diagram. Still, I'd like a review for tech accuracy, especially the parts about the load balancer. I think ChatGPT got it wrong, so I made some changes to that part and wanted to confirm that I did them correctly.